src/bun.js/api/bun/h2_frame_parser.zig

```diff
@@ -657,7 +657,7 @@ pub const H2FrameParser = struct {
     const RefCount = bun.ptr.RefCount(@This(), "ref_count", deinit, .{});
     pub const ref = RefCount.ref;
     pub const deref = RefCount.deref;
-    const ENABLE_AUTO_CORK = true; // ENABLE CORK OPTIMIZATION
+    const ENABLE_AUTO_CORK = false; // ENABLE CORK OPTIMIZATION
```

Comment:
```suggestion
    const ENABLE_AUTO_CORK = true; // ENABLE CORK OPTIMIZATION
```
We need corking otherwise performance will drop a lot